Core/NoOpRunner.Core/PlayerStates/PlayerOneStateMachine.cs

```diff
@@ -14,6 +25,15 @@ public class PlayerOneStateMachine
 
         public bool IsTurnedLeft { get; private set; }
 
+        public bool IsLookingLeft
+        {
+            set
+            {
+                isLastDirectionLeft = IsTurnedLeft;
+                IsTurnedLeft = value;
+            }
+        }
```

Comment:
Can the player be turned left and not looking left? If not, don't have too props. Add a backing field instead if need be.